70_FLIPFluids/app_resources/compute/radix_sort/buildHistogram.comp.hlsl

```diff
@@ -0,0 +1,53 @@
+#include "sort_common.hlsl"
+
+using namespace nbl::hlsl;
+
+[[vk::binding(0, 1)]]
+cbuffer SortParams
+{
+    SSortParams params;
+};
+
+[[vk::binding(1, 1)]] RWStructuredBuffer<DATA_TYPE> inputBuffer;
+[[vk::binding(2, 1)]] RWStructuredBuffer<uint> globalHistograms;
+[[vk::binding(3, 1)]] RWStructuredBuffer<uint> partitionHistogram;
+
+groupshared uint localHistogram[NumSortBins];
```

Comment:
your local histogram was so small that you could have prefix summed it in shared memory with a workgroup right away just like our counting sort example.

anyway with a  Global sort over the whole image, that won't be possible anymore